### DISCUSSION TITLE
Multiple Windows

### DISCUSSION THREAD

**Discussion by hectorregehr** [artisan-roaster-scope/artisan#712] [Q&A] [answered]
I don't know what to call this. A couple years ago I could open as many .alog profiles in as many artisan windows as I wanted to, while artisan was recording a new roast. Now I can only open one window at a time, either viewing a completed recording or watching the live recording, but not together. I find the compare tool confusing to my small brain and can't seem to find info on how others deal with this. I normally like to have the profile I want to follow open to compare with my live recording to see what adjustments I might need to make. Can anyone help me out, please

**Comment by MAKOMO** [comment, marked as answer]
You can open a second instance of Artisan (however not a third). This second instance is somewhat restricted (it cannot record among a view other things) and named ArtisanViewer. On macOS you need to make a copy of the Artisan app and rename it or place it in a different location. Now you can open both of those copies. The second one you open will start in ViewerMode.

Note that the Comparator serves a different purpose. It allows you to compare more than just two profiles in contrast to the standard view where you only have a foreground and background profile.+

See also here: https://artisan-roasterscope.blogspot.com/2020/06/working-together-artisan-artisanviewer.html
